tasks/update-component-sbom/update-component-sbom.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       description: The workspace where the SBOM files reside.
   steps:
     - name: update-component-sbom-purls
-      image: quay.io/konflux-ci/release-service-utils:e15023773e138c7d3449f288c8c16de4e3a8d249
+      image: quay.io/konflux-ci/release-service-utils:3e107dfec5342ef61ae89337458d26c6efe97938
```

Comment:
This is the updated image that contains https://github.com/konflux-ci/release-service-utils/pull/333 , right? Could you please update also the test reference of the image? We like to keep all references within a task folder in sync.

Reply:
Or maybe the other task needs the updated image as well?

Reply:
I updated the tags to use changes from https://github.com/konflux-ci/release-service-utils/pull/342.

Reply:
And populate-release-notes does not need that image.

Reply:
ok, thanks